source/_posts/2025-03-05-release-20253.markdown

```
@@ -0,0 +1,598 @@
+Another nice addition is that the tile card can now be interacted with
+by using your keyboard as well!
+
+<video controls>
+  <source src="/images/blog/2025-03/temp-tile-card-keyboard-navigation.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Editor improvements
+
+Not just the tile card itself has been improved, but also the editor for
+the tile card has been improved. It is now more clear and easier to use. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)